src/huggingface_hub/commands/delete_cache.py

```diff
@@ -120,11 +120,23 @@ def register_subcommand(parser: _SubParsersAction):
             ),
         )
 
+        delete_cache_parser.add_argument(
+            "--sort",
+            nargs="?",
+            choices=["descending", "ascending"],
+            const="descending",
+            help=(
+                "Sort repositories by size. Defaults to descending when flag is used without value. "
+                "Choices: descending (default), ascending"
+            ),
+        )
```

Comment:
```suggestion
        delete_cache_parser.add_argument(
            "--sort",
            choices=["size"],  # TODO: add alphabetical, lastUpdated, lastUsed 
            const="descending",
            help="Criteria to sort repositories",
        )
```

In the future we might want to sort by other criterias (typically `lastUsed` for instance). Therefore, I'd prefer `sort` to be an enum. No need to implement other criteria in this PR, you can continue with `"size"` only.

Also, I don't think we need a direction (ascending or descending). We will assume that users wanting to sort repos by size want to do it in descending order. If in the future we see demand for  sorting by ascending order (which I doubt), I'd rather add another flag `--direction`. But no need to implement it if not requested :)